CHANGELOG-2.0.md

```diff
@@ -0,0 +1,18 @@
+# Changelog since v1.2.0
+
+## New Features
+
+- Add prometheus metrics to CSI external-snapshotter under the /metrics endpoint. This can be enabled via the "--metrics-address" and "--metrics-path" options. ([#227](https://github.com/kubernetes-csi/external-snapshotter/pull/227), [@saad-ali](https://github.com/saad-ali))
+- Split the external-snapshotter sidecar controller into two controllers, a snapshot-controller and an external-snapshotter sidecar. Only the external-snapshotter sidecar should be deployed with the CSI driver. ([#182](https://github.com/kubernetes-csi/external-snapshotter/pull/182), [@xing-yang](https://github.com/xing-yang))
+- Add a finalizer on VolumeSnapshot object to protect it from being deleted when it is bound to
+ a VolumeSnapshotContent. ([#182](https://github.com/kubernetes-csi/external-snapshotter/pull/182), [@xing-yang](https://github.com/xing-yang))
+
+### API Changes
+
+- Changes VolumeSnapshot CRD version from v1alpha1 to v1beta1. ([#139](https://github.com/kubernetes-csi/external-snapshotter/pull/139), [@xing-yang](https://github.com/xing-yang))
```

Comment:
Should this also go under breaking changes since alpha is no longer supported

Reply:
Sure